pr2eus/pr2-interface.l

```diff
@@ -421,8 +421,8 @@ Example: (send self :gripper :rarm :position) => 0.00"
      ))
   (:angle-vector-sequence
    (avs &optional (tms (list 3000)) &rest args)
-   (unless (or (send self :simulation-modep) (cadr (memq :end-coords-interpolation args)))
-     (let* ((prev-av (send robot :angle-vector (send self :state :reference-vector)))
+   (unless (cadr (memq :end-coords-interpolation args))
+     (let* ((prev-av (send robot :angle-vector (if (send self :simulation-modep) (send self :state :potentio-vector) (send self :state :reference-vector))))
```

Comment:
This did not existing tests

may be 
```
   (unless (or (send self :simulation-modep) (cadr (memq :end-coords-interpolation args)))
     (let* ((prev-av (send robot :angle-vector (if (send self :simulation-modep) (send self :state :potentio-vector) (send self :state :reference-vector))))
```
??? please pass tests @Naoki-Hiraoka 
https://github.com/jsk-ros-pkg/jsk_pr2eus/actions/runs/3317672989/jobs/5480786114
```
  + catkin_test_results --verbose --all build
  Full test results for 'pr2eus_moveit/test_results/pr2eus_moveit/rosunit-test_pr2eus_moveit_sim.xml'
  -------------------------------------------------
  <testsuite tests="2" failures="1" disabled="0" errors="0" time="0" name="AllTests">
    <testcase name="all-test" status="run" time="0" classname="all-test">
     <failure message="number expected" type="AssertionError">
  Test:(/ (elt tms l) div)
  Trace:32
  Message:"number expected"
     </failure>
    </testcase>
    <testcase name="test-angle-vector-motion-plan" status="run" time="0" classname="test-angle-vector-motion-plan">
    </testcase>
  </testsuite>
```